runtime/src/qio/qbuffer.c

```diff
@@ -32,6 +32,7 @@
 #include <limits.h>
 #include <sys/mman.h>
 
+#include <ctype.h>
```

Comment:
This isn't needed in the qbuffer header file because of the extern "c" declaration at the top of that file?

This reference that I found on the iovec struct says it is defined in sys/uio.h, why is ctype the one that is added?